Tasks/AzureWebPowerShellDeployment/Publish-AzureWebDeployment.ps1

```diff
@@ -145,10 +145,10 @@ if($azureWebSite) {
             }
         }
         
-        $deploymentId = Get-TaskVariable $distributedTaskContext "build.sourceVersion" #let's use commitId as unique deploymentId in build context
+        $deploymentId = Get-TaskVariable $distributedTaskContext "release.releaseUri" #let's use releaseUri as unique deploymentId in release context
         if([string]::IsNullOrEmpty($deploymentId)) {
-            $deploymentId = Get-TaskVariable $distributedTaskContext "release.releaseUri" #let's use releaseUri as unique deploymentId in release context
-        }        
+            $deploymentId = Get-TaskVariable $distributedTaskContext "build.sourceVersion" #let's use commitId as unique deploymentId in build context
```

Comment:
I like that suggestion, that way if you run the build multiple times on the same sourceversion, we'll still get unique deploymentId's.  I'll make that change ASAP.